docs/developer-guide/syntax-reference.rst

```diff
@@ -306,6 +306,22 @@ defines a struct with three fields, with the supplied name and type.
 Access the fields of a struct by using the ``->`` operator. For example, ``SOME_STRUCT->ID``
 retrieves the value of the struct's ``ID`` field. For more information, see :ref:`operators`.
 
+Decimal
+-------
+
+``DECIMAL(Precision, Scale)``
+
+KSQL supports fields that are numeric data types with fixed precision and scale:
+
+- **Precision** is the maximum total number of decimal digits to be stored, including values to
+  the left and right of the decimal point. The precision must be greater than 1 - there is no
+  default precision.
+- **Scale** is the number of decimal digits to the right of the decimal points. This number must
+  be greater than 0 and less than the value for Precision.
+
+Converting from decimal data type to any floating point type (``DOUBLE``) may cause loss of
+precision. Converting from decimal data type to either ``INTEGER`` or ``BIGINT`` may cause a
```

Comment:
How would the user convert a decimal to an integer?  Are you talking about a `CAST`? There's no implicit conversion. Worth explaining.

Reply:
I will just remove this line altogether - it's probably more misleading than value that it adds.